script/setup/install-critools

```diff
@@ -21,7 +21,7 @@
 set -eu -o pipefail
 
 go get -u github.com/onsi/ginkgo/ginkgo
-CRITEST_COMMIT=16911795a3c33833fa0ec83dac1ade3172f6989e
+CRITEST_COMMIT=2bf7674922a424337d7580a08166d666c6802818 # v1.18.0-100-g2bf7674
```

Comment:
What's the release policy for cri-tools? Is it released in tandem with kubernetes releases?

Reply:
yes https://github.com/kubernetes-sigs/cri-tools/releases